doc/introduction.rst

```diff
@@ -71,6 +71,7 @@ Projects using PyGithub
 * https://github.com/gomesfernanda/some-github-metrics - Python functions for relevant metrics on GitHub repositories
 * https://github.com/SOM-Research/Gitana - a SQL-based Project Activity Inspector
 * https://github.com/YakDriver/satsukia - Python package that helps manage GitHub releases and release assets
+* `check-in <https://github.com/webknjaz/check-in>`_ (`pip install <https://pypi.org/p/check-in>`_) — Python CLI distribution that allows user to use GitHub Checks API as a bot.
```

Comment:
For consistency, would you mind removing the `pip install` part?

Reply:
okay